src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -46,614 +31,1423 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
+            @Config.LangKey("config." + MOD_ID + ".general.food.decayModifier")
+            public double decayModifier = 1.0;
+
+            @Config.Comment("The number of hours to which initial food decay will be synced. When a food item is dropped, it's initial expiration date will be rounded to the closest multiple of this (in hours).")
+            @Config.RangeInt(min = 1, max = 48)
+            @Config.LangKey("config." + MOD_ID + ".general.food.decayStackTime")
+            public int decayStackTime = 6;
+
+            @Config.Comment("If false, crops will never die under any circumstances. THIS DOES NOT MEAN THEY WILL ALWAYS GROW!")
+            @Config.LangKey("config." + MOD_ID + ".general.food.enableCropDeath")
+            public boolean enableCropDeath = true;
+
+            @Config.Comment("Defines wild crops rarity to generate, in 1 / N chunks. 0 = Disable")
+            @Config.RangeInt(min = 0)
+            @Config.LangKey("config." + MOD_ID + ".general.food.cropRarity")
+            public int cropRarity = 30;
+
+            @Config.Comment("Defines berry bush rarity to generate, in 1 / N chunks. 0 = Disable")
+            @Config.RangeInt(min = 0)
+            @Config.LangKey("config." + MOD_ID + ".general.food.berryBushRarity")
+            public int berryBushRarity = 80;
+
+            @Config.Comment("Defines fruit tree rarity to generate, in 1 / N chunks. 0 = Disable")
+            @Config.RangeInt(min = 0)
+            @Config.LangKey("config." + MOD_ID + ".general.food.fruitTreeRarity")
+            public int fruitTreeRarity = 80;
+
+            @Config.Comment("Modifier for how long crops take to grow.")
+            @Config.RangeDouble(min = 0.01, max = 100)
+            @Config.LangKey("config." + MOD_ID + ".general.food.cropGrowthTimeModifier")
+            public double cropGrowthTimeModifier = 1.0;
+
+            @Config.Comment("Modifier for how long berry bushes take to grow fruits.")
+            @Config.RangeDouble(min = 0.01, max = 100)
+            @Config.LangKey("config." + MOD_ID + ".general.food.berryBushGrowthTimeModifier")
+            public double berryBushGrowthTimeModifier = 1.0;
+
+            @Config.Comment("Modifier for how long fruit trees take to grow trunks / leaves / fruits.")
+            @Config.RangeDouble(min = 0.01, max = 100)
+            @Config.LangKey("config." + MOD_ID + ".general.food.fruitTreeGrowthTimeModifier")
+            public double fruitTreeGrowthTimeModifier = 1.0;
+        }
+
+        public static final class MiscCFG
+        {
+            @Config.Comment("The default length of a month (in days) when a new world is started. This can be changed in existing worlds via the /timetfc command.")
+            @Config.LangKey("config." + MOD_ID + ".general.misc.defaultMonthLength")
+            public int defaultMonthLength = 8;
+
+            @Config.Comment("Percentage chance that plants (not crops) grow each update. Smaller number = slower growth.")
```

Comment:
This needs to be explained better.
```suggestion
            @Config.Comment("Chance for a plant to grow each random tick, does not include crops. Lower = slower growth."")
```